### DISCUSSION TITLE
TypeHelpers for Cloudscape

### DISCUSSION THREAD

**Discussion by bishopZ** [cloudscape-design/components#2246] [Show and tell]
Hey Team,

I've been using Cloudscape for a few years now and I must say, you guys have done an amazing job. For instance, the Typescript definitions are top-notch – comprehensive and easy to navigate. However, I've noticed that some of those type names are quite long. To make things a bit more readable in my application, I created some type helpers.

https://bishopz.com/articles/type-helpers

These helpers are all about providing shorthand interfaces for various Cloudscape components within my app. They don't replace or change the official definitions – instead, they serve as a central hub for shorter type names that can be shared across components. I think the shorter names make our code more readable and easier to grasp.

I wrote an article about my experience using TypeHelpers with Cloudscape, and I'd love to share this knowledge with the rest of the developer community! My website doesn't get much traffic, and my goal is to contribute back to the ecosystem and make development with Cloudscape even more enjoyable for everyone.

Do you have any suggestions or ideas on how best to spread the word about TypeHelpers? 

Any help appreciated!

**Comment by just-boris** [comment]
Thanks for sharing your use-case with us!
